### ISSUE TITLE
Use different icons for SingleFileZ

### ISSUE THREAD

**Issue by gildas-lormeau** [gildas-lormeau/SingleFileZ#14]
I tried to transform the existing ones with Gimp by using filters but it was not pretty at all. Suggestions or contributions are welcome :)

**Comment by kramola-RU** [comment]
Could you make separate SFZ-icon -- visually different from the SF-icon ?

**Comment by gildas-lormeau** [comment]
Yes I could :)

**Comment by gildas-lormeau** [comment]
FYI, the choice for the current icon was made here: https://github.com/gildas-lormeau/SingleFile/issues/137

**Comment by kramola-RU** [comment]
I think it should be icon with a **Z** letter (or SZ - unlike **Z**oom)

**Comment by gildas-lormeau** [comment]
You mean like **Z**ip? :p

**Comment by kramola-RU** [comment]
yep  :)

**Comment by gildas-lormeau** [comment]
What about using this one? https://www.flaticon.com/premium-icon/zip_236107

**Comment by gildas-lormeau** [comment]
What about an icon that would be similar to a zip icon? I can't find one right now that would do the job but it should be possible to find one.

**Comment by kramola-RU** [comment]
why not... it will be pretty :)

**Comment by gildas-lormeau** [comment]
If, by pure coincidence, you find one (free) that you like, do not hesitate to suggest it ;)

**Comment by kramola-RU** [comment]
deal
but I'm not sure about my artistic taste :)

-------------------------
one more a little Q

why are you using the:
"id": "{e4db92bc-3213-493d-bd9e-5ff2afc72da6}"
in the manifest?
why not the "id": "SingleFileZ@seo4ajax" ?

**Comment by gildas-lormeau** [comment]
Thank you :)

Regarding the ID, I've never thought it would be useful to customize it. I would not use seo4ajax.com  anyway because it's not related with SingleFile/SingleFileZ.

**Comment by kramola-RU** [comment]
there may be any text after @ (it is not necessary to use the seo4ajax.com)

regarding any advantage... I like to see readable addon-name in the folder:
[img]https://www.upload.ee/image/10299056/123.png[/img]

**Comment by gildas-lormeau** [comment]
Thanks for the info. I'll see how to customize it.

Note that I can indeed translate your discussions from https://forum.ru-board.com (it seems the icon problem is more "important" than I  was thinking). However, the translation from Google is quite awful and I don't speak Russian at all. It would be easier for me if you could post your requests/suggestions here.

**Comment by gildas-lormeau** [comment]
What about this icon? https://www.flaticon.com/free-icon/file_1570432

**Comment by kramola-RU** [comment]
yes
I like this

**Comment by gildas-lormeau** [comment]
I like it too. I will change it as soon as I have some time to do so.

**Comment by gildas-lormeau** [comment]
Note that I do have the right to modify icons. It must be used as is. I'm referring to the propositions on the forum ;)

**Comment by kramola-RU** [comment]
OK
thank you very much for your addons

**Comment by harryks** [comment]
Icon variant for SFZ
![sfz_zw128](https://user-images.githubusercontent.com/42656194/62306318-4e150e00-b48a-11e9-9d6f-095c44d748dc.png)

**Comment by gildas-lormeau** [comment]
@harryks As I said, I cannot modify existing icons. I do not have the right to do so. I also just published the version 0.2.16 with the new icons. Unfortunately, it looks like Firefox is buggy and you may see the 2 icons (the old one and the new one) mixed for a short period of time.

**Comment by harryks** [comment]
I do not understand. You do not have the right to your own icons? The addon belongs to you, but the icons do not belong to you? Okay, let’s leave it.
SingleFileZ exports the settings to a file called singlefile-settings.json. I think it's better to add the letter "z" - singlefilez-settings.json, so that it differs from the SingleFile settings.

**Comment by gildas-lormeau** [comment]
It's not my icons, I don't have the intellectual property. These are icons provided by flaticon. However, I was wrong when I was telling you that I did not have the right to modify them. I'll rename the settings file as you've suggested.

**Comment by gildas-lormeau** [comment]
I renamed the exported settings file to singlefilez-settings.json in the version 0.2.17 I just published. I'm closing this issue. Feel free to re-open it if necessary.
